packs/fastly/config.yml

```diff
@@ -0,0 +1,46 @@
+# Name of the pack (required)
+name: fastly-cdn
+
+# Displayed in the UI (required)
+title: Fastly CDN
+
+# Long-form description of the pack (required)
+description: |+
+  Fastly helps people stay better connected with the things they love. Fastly’s edge cloud platform enables customers to create great digital experiences quickly, securely, and reliably by processing, serving, and securing our customers’ applications as close to their end-users as possible — at the edge of the internet. Fastly’s platform is designed to take advantage of the modern internet, to be programmable, and to support agile software development with unmatched visibility and minimal latency, empowering developers to innovate with both performance and security. Fastly’s customers include many of the world’s most prominent companies, including Pinterest, The New York Times, and GitHub.
+
+# Displayed in search results and recommendations. Summarizes a packs functionality.
+summary: |
+  Analyze the health of your Fastly CDN POP footprint with both a dashboard and alerts.
+
+# Support level: New Relic | Verified | Community (required)
+level: Verified
+
+# Authors of the pack (required)
+authors:
+  - Simon Wistow (Fastly) & Josh Biggley (New Relic)
```

Comment:
Let's separate in 2 lines (authors is a list)

Reply:
Done